### DISCUSSION TITLE
Unity SDK Update 2.7.1

### DISCUSSION THREAD

**Discussion by YVRDevelopers** [PlayForDreamDevelopers/support#9] [Announcements]
**新增：
Core**
- 新增**超采样**和**锐化功能**，提高图像质量。超采样适用于解决下采样时的闪烁及摩尔纹等问题。锐化改善上采样时的图像模糊问题。
- 支持**动态注视点渲染**，系统会根据当前 GPU 的使用情况动态地调整注视点渲染级别。
- 新增**手柄震动时长**接口，支持让手柄连续震动一段时间。
- 支持调整**屏幕刷新率**，可根据需要调整屏幕每秒的刷新次数。

**修复:
Core**
- 修复强制依赖手势追踪的应用无法获取手柄数据问题
**下载地址：** https://developer.yvr.cn/yvrdoc/sdk/unity/UnitySDK_2_7_1.zip

**Added:
Core**
- Added **supersampling** and **sharpening** capabilities to enhance image quality. supersampling helps address flickering and moiré patterns during downsampling. Sharpening can improve clarity and alleviate image blurring during supersampling.
- Added support for **dynamic foveated rendering**, allowing the system to adjust foveation rendering levels dynamically based on current GPU usage.
- New interface for **controlling duration of controller vibration**, enabling continuous vibration for a specific duration.
- Added support for adjusting screen refresh rate, allowing adjustment of the number of screen refreshes per second as needed.

**Fixed:
Core**
- Fixed issue where applications relying on hand tracking were unable to obtain controller data.
**Download Link:** https://developer.yvr.cn/yvrdoc/sdk/unity/UnitySDK_2_7_1.zip

**Comment by Igloo302** [comment]
Nice Job
